### ISSUE TITLE
vs 36 sounds 

### ISSUE THREAD

**Issue by chase911** [skiselkov/BetterPushbackC#76]
I downloaded vs. 36. And everything worked. But it muffled all sounds.  with the sliders all the way up i noticed still at least a 35% reduction in sound volume. Engines environment SQuawkbox PE everything. I removed the plugin and problem persisted. I then did a xplane update thinking that would overwrite anything your coding did to the sound coding of xplane...still no fixing the problem. i had to restore to a earlier image to fix the sounds since i didn't know what else to do. That took hours. But I'm back in business. I'm hard of hearing so i really need to be able to hear what pilotedge controllers are saying.  Even though it was really cool.  I'm scared to download again untill you have a finished product. But i know in beta there will be issues and without my remarks you won't know what to fix.  FF 767-300er was just updated and their update said compatible with vs 29 and above. That's where i heard about your awesome free tug. Do you think vs 29 would mess with my xplane master volumes? Thanks for all the work your putting into this project for the community. 
Dave.

**Comment by skiselkov** [comment]
Thanks for the bug report. I'm not sure how the plugin could be muffling sounds in any way. It never writes to any sound volume settings, it only reads them. Could you please make a video of the effect happening (i.e. record one video before with the sound as normal, second with the plugin in place and the sounds muffled). In any case, you should be able to restore X-Plane's preferences by simply backing up and restoring the `Output/preferences` folder. No need to do full simulator reinstall or restore.

**Comment by chase911** [comment]
Were  you ever able to diagnose a problem with the affected sounds?  Ie your most current release would not lower the over all global sounds of everything?  Does this plugin have its own sound volume control slider by any chance? I thought about that afterword.  That maybe there was a slider that i could of moved that would of fixed the issue. Or does it use the xplane's default sound settings? Thanks.  I'm hoping to download and give it another try. ; )
Dave.

**Comment by skiselkov** [comment]
I asked you to make a video of the problem, because from your description I couldn't see how BP would or could be affecting the volume of other parts of X-Plane. BP **NEVER** writes to any sound volume settings, it only ever reads them. Anyway, the latest version uses three volume settings from X-Plane (11):
1. the Exterior volume slider controls tug engine & mechanism loudness when in an outside view or when the cockpit window is opened.
1. the Interior volume slider controls tug engine & mechanism loudness when in an internal view and when the cockpit windows are closed.
1. the Radio volume slider controls the loudness of the ground crew audio.

Without a video of the effect you described of the sound mysteriously going quiet I can't help you. You're the only person reporting such a problem and I obviously couldn't reproduce any such problem on my system (otherwise I would have fixed it long ago).

**Comment by chase911** [comment]
Ok I'm sorry. I didn't want to re-download the same Version.  And maybe it was just a fluke. Maybe it had nothing at all do to with your TUG. I was only reporting in good faith what I believed had caused a problem.  I will Download your latest version and will let you know how that turns out  I will make a quick recording of my sounds before I do.  Thanks for all the hard work you are putting into this plugin. Dave. 

    On Thursday, August 10, 2017 10:36 AM, Saso Kiselkov <notifications@github.com> wrote:
 

 I asked you to make a video of the problem, because from your description I couldn't see how BP would or could be affecting the volume of other parts of X-Plane. BP NEVER writes to any sound volume settings, it only ever reads them. Anyway, the latest version uses three volume settings from X-Plane (11):   
   - the Exterior volume slider controls tug engine & mechanism loudness when in an outside view or when the cockpit window is opened.
   - the Interior volume slider controls tug engine & mechanism loudness when in an internal view and when the cockpit windows are closed.
   - the Radio volume slider controls the loudness of the ground crew audio.
Without a video of the effect you described of the sound mysteriously going quiet I can't help you. You're the only person reporting such a problem and I obviously couldn't reproduce any such problem on my system (otherwise I would have fixed it long ago).—

**Comment by skiselkov** [comment]
Thanks and I do appreciate the help! Don't worry, we'll get this sorted.

**Comment by chase911** [comment]
OK I just wanted to keep you up to date. So last night I backup everything I had...So I was not scared to try anything today lol. But anyway, I Checked my sounds before  downloaded your current .41 Version.  Then I checked the sounds after.  No problems found.  I then did a flight, then exited the SIM. Restarted and tried two  different aircraft and still no problems.  So I would go ahead and close this ticket as fixed or never was a problem.  I guess it was a fluke. But regardless your most up to date version works like a charm.  Thanks. Dave.  

    On Thursday, August 10, 2017 11:30 AM, Saso Kiselkov <notifications@github.com> wrote:
 

 Thanks and I do appreciate the help! Don't worry, we'll get this sorted.—
